src/en/space-station-14/player-interaction/proposals/New Antagonist: The Spider Queen

```diff
@@ -0,0 +1,48 @@
+# New Antagonist: The Spider Queen
+
+Adds a new antagonist: The spider queen
+
+| Designers | Implemented | GitHub Links |
+|---|---|---|
+| FraG/Dmitry777 | :information_source: Open PR | https://github.com/space-wizards/space-station-14/pull/30820 |
+
+## Overview
+
+Adds a new antagonist who looks a bit like the rat king. More details in PR
```

Comment:
Please put the details on the PR onto the design document.

Reply:
If I'm not mistaken, it should be described in 3 sentences.

Reply:
Yes, but instead of putting most of the information on the PR, it should be copied entirely across (below the overview) over to the document. It might help to think of it as seperate to the PR, nothing more than saying it’s vaguely related.